pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -22,14 +21,12 @@ def get_treeinfo_data(remote, remote_url):
     treeinfo_serialized = {}
     namespaces = [".treeinfo", "treeinfo"]
     for namespace in namespaces:
-        downloader = remote.get_downloader(url=urljoin(remote_url, namespace))
+        downloader = remote.get_downloader(
+            url=urljoin(remote_url, namespace), silence_errors_for_response_status_codes={404}
```

Comment:
it explicitly silences for 404, so I don't think we should raise `FileNotFoundError` for 4xx